src/dns-lookup/templates/app.vue

```diff
@@ -76,8 +78,10 @@ limitations under the License.
     import RecordBase from "./record_base"
     import RecordJumps from "./record_jumps"
     import i18n from "../i18n"
+    import i18nShared from "../../shared/i18n"
```

Comment:
Is this the best way to access the shared i18n data?

Reply:
Yeah, I think so personally.